docs/dev/table/tableApi.md

```diff
@@ -101,25 +101,49 @@ val result = orders
 </div>
 <div data-lang="python" markdown="1">
 
-The Python Table API is enabled by `from pyflink.table import *`.
-
 The following example shows how a Python Table API program is constructed and how expressions are specified as strings.
 
 {% highlight python %}
 from pyflink.table import *
-from pyflink.dataset import *
 
 # environment configuration
-env = ExecutionEnvironment.get_execution_environment()
-t_env = TableEnvironment.create(env, TableConfig())
+t_env = BatchTableEnvironment.create(
+    environment_settings=EnvironmentSettings.new_instance().in_batch_mode().use_blink_planner().build())
 
 # register Orders table and Result table sink in table environment
-# ...
+source_data_path = "/path/to/source/directory/"
+result_data_path = "/path/to/result/directory/"
+source_ddl = """
```

Comment:
It would be better if we use the format strings:
f"""
           create table Orders(
            a VARCHAR,
            b BIGINT,
            c BIGINT,
            rowtime TIMESTAMP(3),
            WATERMARK FOR rowtime AS rowtime - INTERVAL '1' SECOND
        ) with (
            'connector' = 'filesystem',
            'format' = 'csv',
            'path' = '{source_data_path}'
        )
 """